### ISSUE TITLE
Problems with special characters in latest update?

### ISSUE THREAD

**Issue by rubjm9** [benhuson/wp-subtitle#22]
Hello, these past few days we have been having some troubles with special characters in our WordPress posts' subtitles, particularly with some of the Spanish á é í ó ú, which sometimes (but not always) appear as Ã³ and similar.

Since there has been a plugin update recently, I was wondering if it could be related, or there was any known bug.

A couple examples:
http://www.somosbasket.com/2016/09/17/noticias/masai-ujiri-toronto-es-mi-hogar/
http://www.somosbasket.com/2016/09/17/noticias/joel-embiid-no-back-to-back/

Thanks! (:

**Comment by benhuson** [comment]
Hi, not a bug I am aware of and not sure how that anything in the recent update could cause this.

I am a bit confused by your "which sometimes (but not always)" comment. Does this mean that sometimes those characters save OK and sometimes they don't?

Does the issue happen when you save posts or just when viewing the site?

If you have multiple post types, does it happen just on some post types and not others?

Thanks

Ben

**Comment by rubjm9** [comment]
I think I figured out when it happens. 

So we publish an article that has a subtitle with these Spanish special characters (á é í ó ú). At first the article's subtitle is published perfectly fine, BUT if you try and edit the article, these characters appear switched into weird combinations such as Ã³, etc.

Then we edit the article and when we update it, these characters replace the originals in the public version of the entries.

Another curious thing happens then: whenever we try and edit an article with one of these weird characters, the subtitle gets erased in the edition view of the article! 

To summarize: This means that if we save changes not double checking the subtitle every time, the first time we save changes the special characters are switched into weird letters; and if we edit an article with weird letters, the subtitle gets erased.

This has been happening for the past few days, before it used to work fine. I don't know whether it is related or not to the plugin's latest update, but it seems to coincide in the time. I hope this research is helpful to find the bug, let me know if I wasn't clear about anything.

Thanks!

Rubén

**Comment by benhuson** [comment]
@rubjm9 Please try download this branch of the plugin with a fix that may resolve the issue:

https://github.com/benhuson/wp-subtitle/archive/hotfix/html-entities-admin-field.zip

Rename the download plugin folder to `wp-subtitle` and replace the plugin on your site.

Please let me know if this works for you.

Ben

**Comment by marcospulido** [comment]
@benhuson works for me. Thnaks!

**Comment by hatsumatsu** [comment]
Hi there, just chiming in to ask if this change could be included in the master / WP plugin repo.

**Comment by benhuson** [comment]
Sorry, yes, I'll get it in the next release to WP.org

**Comment by benhuson** [comment]
Fix in https://github.com/benhuson/wp-subtitle/commit/51be9d1854d1a89136f8d5eaa181fa1884a89a11

Will be in next release.
